### PR TITLE
task handin

### PR DESCRIPTION
**Pull request by tehvicke**

### CODE REVIEW

**Comment by tehvicke** [comment]
Oh ok! Only included it in the canvas handin. Here you go! https://eloquent-wozniak-2eeff2.netlify.com/

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <title>WeatherApp</title>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <!-- <header>
+        <h1>WeatherApp</h1>
+    </header> -->
+    <div class="rounded-bg"></div>
```
Comment (ericaglimsholt) [inline code/index.html]: Are you using this in any way?

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        </section>
+        <section class="weather-forecast">
+            <ul id="forecast-weather-list" class="forecast-list">
+            </ul>
+
+        </section>
+    </main>
+    <footer>
+        <!-- <p>WeatherApp Footer</p> -->
+    </footer>
```
Comment (ericaglimsholt) [inline code/index.html]: Why have you comment out this?

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    "Mon",
+    "Tue",
+    "Wed",
+    "Thu",
+    "Fri",
+    "Sat",
+    
+]
+
+// A  weather URL is made up of xxx_URL + cityName + URL_SUFFIX. Should use country code to avoid duplicate cities but let's keep it simple for now...
```
Comment (ericaglimsholt) [inline code/script.js]: Would be fun to see if you implement this later on!

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+
+// add test attributes
```
Comment (ericaglimsholt) [inline code/script.js]: ?

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+    }).then((json) => {
+        console.log("2.")
+        console.log(json)
+    }).catch ((err) => {
+        console.error("Error.", json)
+    })
+    console.log("Finished testFunction.")
+}
+//testFunction(FORECAST_URL, cityName)
+//testFunction(CURRENT_URL, cityName)
```
Comment (ericaglimsholt) [inline code/script.js]: Great that you are using test functions in the way you work, but if you are not using this function you can delete it from your completed project

**Pull request closed without merging**